lib/codegen/fromcto/golang/golangvisitor.js

```diff
@@ -169,6 +169,12 @@ class GoLangVisitor {
             array = '[]';
         }
 
+        if(this.isMap(field)) {
+            const decl = field.getModelFile().getAllDeclarations().find(d => d.name === field.ast.type.name);
```

Comment:
Do we have a function that does this? e.g `field.getName()`, `field.getShortName()` instead of `field.ast.type.name`

Reply:
`field.getType()` returns the unqualified name of the field or `field.getFullyQualifiedTypeName()`.

Reply:
Assumes the type of the map value is in the same model as the map decl.